tendrl/commons/utils/ssh_util.py

```diff
@@ -0,0 +1,64 @@
+import psutil
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.commons.utils import cmd_utils
+
+
+def sshd_status():
```

Comment:
This util must always return a specific data structure, which would seem to be a dict with keys "name", "port" and "status". This must always be the return value. If there are any errors, the values should probably be Null or empty, whatever is the appropriate python convention. In none of the error conditions are you supposed to return directly with just a string. Use log messages to indicate errors.